### PR TITLE
Student history

### LINKED ISSUE #116: PollHistory: Student answer history
**Please describe what has to be done**

- [x] Design Complete/Approved
- [x] Code Created
- [x] Code approved/merged

**Additional context**

Create a new page that shows a list of all polls that you've answered (grouped by group if applicable) and those link to the pollresults page for each. 

Accessed thru header button (see https://github.com/PollBuddy/PollBuddy/issues/174)

### PR DESCRIPTION
**Pull request by toneofgrey**
### Description of change(s), including why:
(Created PollsHistory page and added functionality for admins and members of a poll to access poll results, and if user is not in a poll, display message instead)

### Type of change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

- (The only thing left to do in this page is to dynamically grab user info from backend and use that, otherwise the other issue are with other pages such as PollsResults functionality).

### Closing issues:

- Closes #116 

### Screenshots (if frontend change) of before and after:
Default
empty

### CODE REVIEW

File: PollBuddy-Server/frontend/src/App.js [hunk 1]
```diff
@@ -148,6 +149,10 @@ export default class App extends React.Component {
               <Template updateTitle={this.updateTitle} />
             </Route>
 
+            <Route exact path="/polls/:pollID/history">
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/App.js:152]: Probably better as just /polls/history. Also can you move it up with the other polls routes, probably at line 111?

File: PollBuddy-Server/frontend/src/pages/PollHistory/PollHistory.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+export default class Groups extends Component {
+  constructor() {
+    super();
+    if (!localStorage.getItem("loggedIn")) {
+      //Redirect("/login");//this is a way to redirect the user to the page
+      //window.location.reload(false);//this forces a reload so this will make the user go to the login page. A little barbaric but it works. If frontend wants to make it better by all means
+    }
+  }
+
+  signout() {
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/PollHistory/PollHistory.js:19]: This is probably not necessary I'd think. I know you were just working off another page so I imagine that's why it's there.

File: PollBuddy-Server/frontend/src/App.js (hunk 1 above)
Comment (hbh7) [reply PollBuddy-Server/frontend/src/App.js:152]: Oops I didn't make this a review comment

**Review by hbh7 — CHANGES_REQUESTED** [review]
Looking good so far, just a few things: 

- [ ] Any reason App.test.js was deleted? 
- [ ] A few lines were commented on
- [ ] Some lint fixes
- [ ] Rename the page title to something like "My Submitted Polls" like we talked about in our group meeting

File: PollBuddy-Server/frontend/src/pages/PollHistory/PollHistory.js (hunk 2 above)
Comment (toneofgrey) [reply PollBuddy-Server/frontend/src/pages/PollHistory/PollHistory.js]: Unused Code deleted.

File: PollBuddy-Server/frontend/src/App.js (hunk 1 above)
Comment (toneofgrey) [reply PollBuddy-Server/frontend/src/App.js]: Moved and changed path and import in App.js

**Comment by toneofgrey** [comment]
For some reason Cl-PullRequest is now breaking

**Comment by hbh7** [comment]
> For some reason Cl-PullRequest is now breaking

Don't worry about it, apparently something broke upstream from us. Nothing I can do for now. Someone already reported it and I'm following their issue. 

Your changes are looking good though, I'll look more closely soon. :D

**Review by nbezanis — APPROVED** [review]

**Review by hbh7 — APPROVED** [review]
Looks good!

**Pull request merged**